<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairSplitWidget</class>
 <widget class="QWidget" name="PairSplitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QWidget" name="topContainer">
      <layout class="QHBoxLayout" name="topLayout">
       <item>
        <widget class="QTableWidget" name="teamATable">
         <property name="columnCount">
          <number>1</number>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="teamBTable">
         <property name="columnCount">
          <number>1</number>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="bottomContainer">
      <layout class="QHBoxLayout" name="bottomLayout">
       <item>
        <widget class="QTabWidget" name="teamATabs">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <widget class="QWidget" name="statusTabA">
          <attribute name="title">
           <string>Status</string>
          </attribute>
          <layout class="QVBoxLayout" name="statusLayoutA">
           <item>
            <widget class="QLabel" name="teamALabel">
             <property name="text">
              <string>Team A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignHCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inventoryTabA">
          <attribute name="title">
           <string>Inventory</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="rlTabA">
          <attribute name="title">
           <string>RL</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="teamBTabs">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <widget class="QWidget" name="statusTabB">
          <attribute name="title">
           <string>Status</string>
          </attribute>
          <layout class="QVBoxLayout" name="statusLayoutB">
           <item>
            <widget class="QLabel" name="teamBLabel">
             <property name="text">
              <string>Team B</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignHCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inventoryTabB">
          <attribute name="title">
           <string>Inventory</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="rlTabB">
          <attribute name="title">
           <string>RL</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
